### PR TITLE
add ui schema property to scaffolder templates

### PR DESCRIPTION
**Pull request by ydayagi**
FLPATH-1615
https://issues.redhat.com/browse/FLPATH-1615

### CODE REVIEW

**Comment by batzionb** [comment]
Adding the ui:autofocus is something that currently the UI already does automatically for the first property of each step
Maybe a better example would be to add

ui:widget: textarea

File: scaffolder-templates/basic-workflow/skeleton/src/main/resources/schemas/${{values.artifactId}}-input-schema.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "sampleText": {
       "title": "A sample text",
       "type": "string",
-      "description": "Sample text data"
+      "description": "Sample text data",
+      "ui:autofocus": true
```
Comment (batzionb) [inline scaffolder-templates/basic-workflow/skeleton/src/main/resources/schemas/${{values.artifactId}}-input-schema.json:11]:
```suggestion
      "ui:widget": "textarea"
```

File: scaffolder-templates/basic-workflow/skeleton/README.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ instance of the ${{ values.workflowId }} workflow:
 curl -v -X POST -H "Content-Type: application/json" http://localhost:8080/${{ values.workflowId }} -d@input.json
 ```
 
+## Adding UI schema properties
+You can add React UI schema properties to the input schema. See the 'ui:autofocus' example at [${{ values.artifactId }}-schema.json](./src/main/resources/schemas/${{ values.artifactId }}-schema.json ).
```
Comment (batzionb) [inline scaffolder-templates/basic-workflow/skeleton/README.md:42]:
```suggestion
You can add React UI schema properties to the input schema. See the 'ui:widget' example at [${{ values.artifactId }}-schema.json](./src/main/resources/schemas/${{ values.artifactId }}-schema.json ).
```

Comment (ydayagi) [reply scaffolder-templates/basic-workflow/skeleton/README.md]: done

**Review by batzionb — APPROVED** [review]

**Review by batzionb — APPROVED** [review]

**Review by masayag — APPROVED** [review]

**Pull request merged**